akvo/iati/checks/fields/budgets.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+
+def budgets(project):
+    """
+    Check if budget has start date, end date and a value.
+    Check that start date lies before the end date.
+    Check if the budget has a currency if there is not default currency.
+
+    :param project: Project object
+    :return: All checks passed boolean, [Check results]
+    """
+    checks = []
+    all_checks_passed = True
+
+    for budget in project.budget_items.all():
+        if budget.amount is None:
+            all_checks_passed = False
+            checks.append((u'error', u'budget (id: %s) has no amount' % str(budget.pk)))
```

Comment:
Use .format(). for string formatting. You get
  * automatic casting of non-string types
  * python3 compatibility :wink: 

(Using Pycharm's regex find/replace I think you can fix this in just a few passes, one for each type of casting)